### DISCUSSION TITLE
Week 1 Xion_Nights

### DISCUSSION THREAD

**Discussion by rastitiratnasari** [teamalgoritma/community#808] [Q&A] [answered]
1. Contoh real penggunaan atribut axes di python
2. Penggunaan `.unique()`  dan .nunique() di python
3. Membaca file excel di python

**Comment by saskia-dwi-ulfah** [comment, marked as answer]
Dear @rastitiratnasari 

1. Atribut `.axes` bisa kita gunakan untuk melihat nama kolom dan baris dari suatu dataframe. Atribut `.axes` akan sangat berguna pada saat kita melakukan exploratory data analysis dan pembersihan data.
2. `.unique()` digunakan untuk **melihat nilai unik** dari suatu kolom. Sementara itu, `.nunique()` digunakan untuk melihat **banyak nilai unik**  dari suatu kolom.
3. Untuk membaca file Excel di Python bisa menggunakan fungsi `pd.read_excel()`.

Semoga jawaban ini dapat membantu.

Best regards,
Saskia Dwi Ulfah

**Comment by kinantytasyao** [comment]
Halo @rastitiratnasari , karena tidak ada konfirmasi lebih lanjut dan untuk menjaga agar forum diskusi tetap berjalan kondusif, kami izin melakukan lock/close diskusi ini. Semoga dapat dipahami.
